### PR TITLE
Add Polish translation (MMH)

### PR DESCRIPTION
**Pull request by Radplay**

### CODE REVIEW

File: Mod Menu Helper/assets/fo/lang/pl_pl.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  "modmenu.summaryTranslation.entity_texture_features": "Różne i emisyjne moby OptiFine'a\n§9✎§8 Różne moby, emisyjność, mruganie oczu, ...",
+  "modmenu.summaryTranslation.entityculling": "Zapobiega renderowaniu modów za ścianami\n§8✎ (Tylko zaawansowane opcje)",
+  "modmenu.summaryTranslation.fabricskyboxes": "Niestandardowe nieba\n§9✎§8 Rozwiń aby skonfigurować (Kliknij na ikonę ▼)",
+  "modmenu.summaryTranslation.fsb-interop": "Wspracie OptiFine'a dla FabricSkyBoxes\n§9✎§8 Wsparcie OptiFine'a/MCPatcher'a/natywne, ...",
+  "modmenu.summaryTranslation.fabrishot": "Rób zrzuty ekranu w HD klikając F9\n§9✎§8 Nadpisz F2, rozdzielczość, skrót klawiszowy, ...",
+  "modmenu.summaryTranslation.farsight": "Utrzymuje załadowane chunki widoczne na serwerach\n§8✎ (tylko zaawansowane opcje)",
+  "modmenu.summaryTranslation.fastchest": "§c*§r Wydajniejsze skrzynie\n§9✎§8 Przełącz mod",
+  "modmenu.summaryTranslation.ferritecore": "Zmniejsza użycie RAM'u\n§8✎ (tylko zaawansowane opcje)",
+  "modmenu.summaryTranslation.holdthatchunk": "§c*§r Utrzymuje załadowane chunki widoczne na serwerach\n§8✎ (tylko zaawansowane opcje)",
```
Comment (Madis0) [inline Mod Menu Helper/assets/fo/lang/pl_pl.json:30]: image
2nd line doesn't fit

**Review by Madis0 — CHANGES_REQUESTED** [review]
Cool! Some translations don't fit on a FullHD screen though, commented below.

**Comment by Radplay** [comment]
Should be fine now

**Review by Madis0 — APPROVED** [review]
Will be merged on the next release

**Pull request merged**